### DISCUSSION TITLE
lens.google.com

### DISCUSSION THREAD

**Discussion by Daxy7** [uBlockOrigin/uAssets#19064] [General]
firefox cache, cookie or site data storage bug when scanning file says no text recongized. and keeps happening. how to perma fix?
any common causes?

![image](https://github.com/uBlockOrigin/uAssets/assets/91396482/f1340f68-7a3a-44aa-85cb-cd62ca8881bb)

**Comment by stephenhawk8054** [comment]
Can you describe steps to reproduce in more details?

**Comment by Daxy7** [comment]
> cache, cookie or site data storage bug 

idk. maybe my drive is dying but the stats say everything's good. i'll try clearing all site data storages and report back in a month or two
(the avg time when it relapses)

edit: this also happened with being signed into my account so not sure if to just start fresh and use another one

**Comment by stephenhawk8054** [comment]
I mean what are the steps to start reproducing the issue. This is what appears for me when accessing `https://lens.google.com`: https://github.com/uBlockOrigin/uAssets/assets/66517106/3311945d-e6f2-4e25-92d7-529413b6203b

**Comment by Daxy7** [comment]
but yes clearing site data worked. some site issue or browser webcompat issue.
```
https://www.google.com/
```
then click 'search by image' camera icon paste this random image from imgur
```
https://i.imgur.com/iA3M3OD.jpg
```
